examples/charts/experimental.htm

```diff
@@ -626,6 +626,94 @@
                 </div>
             </div>
 
+            <div class='row trunk-section'>
+                <div class='col-lg-7 text-center'>
+                    <div class="tooltip"></div>
+                    <style>
+                    .tooltip {
+                        display: none;
+                        min-height: 32px;
+                        min-width: 100px;
+                        padding: 12px;
+                        margin-bottom: 8px;
+                        background-color: #333;
+                        color: white;
+                        opacity: 0.7;
+                        text-align: left;
+                    }
+                    </style>
+                    <div class='col-lg-12 text-center' id='tooltip-line-chart'></div>
+                    <div class='col-lg-12 text-center' id='tooltip-point-chart'></div>
+                </div>
+                <div class='col-lg-5'>
+
+<pre><code class='html'>&#x3C;div class=&#x22;tooltip&#x22;&#x3E;&#x3C;/div&#x3E;
+&#x3C;style&#x3E;
+.tooltip {
+    display: none;
+    min-height: 32px;
+    min-width: 100px;
+    padding: 12px;
+    margin-bottom: 8px;
+    background-color: #333;
+    color: white;
+    opacity: 0.7;
+    text-align: left;
+}
+&#x3C;/style&#x3E;
+</code></pre>
+
+<pre><code class='javascript'>const tooltipEl = d3.select('.tooltip').node();
+const tooltip = new Popper(document.documentElement, tooltipEl, { placement: 'top' });
+
+d3.json('data/fake_users1.json', function(data) {
+    data = MG.convert.date(data, 'date');
+    MG.data_graphic({
+        title: "Tooltip on line chart",
+        description: "This is an example to use popper.js to show a tooltip on line chart.",
```

Comment:
Change this (here and below, too bad we duplicate the code in two places for these examples) to:

`Using popper.js and the mouseover callback, you can easily create a tooltip when the user hovers over a particular datapoint.`